js/src/pages/price-benchmark/price-benchmark-tour.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { TourKit } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import useTour from '~/hooks/useTour';
+import PriceBenchmarkGraphic from '~/images/price-banchmark-tour.svg';
+
+const TOUR_ID = 'price-benchmark-tour';
+
+/**
+ * Renders the tour for notifying the new feature of price benchmark & suggestions.
+ */
+export default function PriceBenchmarkTour() {
+	const { tourChecked, setTourChecked } = useTour( TOUR_ID );
+
+	if ( tourChecked ) {
+		return null;
+	}
+
+	const config = {
+		steps: [
+			{
+				referenceElements: {
+					desktop: '.gla-price-benchmark__card a[id="suggestions"]',
+				},
+				meta: {
+					heading: (
+						<div className="gla-price-benchmark-tour__heading">
```

Comment:
@ankitrox If you inspect the markup on the FE, you'll notice that the `div` is within a `h2` tag. Generally this goes against HTML semantics. We can simplify things by doing the following:

1. Remove the `div`
2. Wrap the `Price Benchmark & Suggestions` text within a `span` tag.
4. Set the styles for `gla-price-benchmark-tour__graphic` to `display: block` and `margin: 0 auto $grid-unit-40` where `$grid-unit-40` is `32px`
5. Set the span styles to `display: block`